lib/assessment.js

```diff
@@ -38,8 +81,38 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // If the argument obj is unset, create a new object
 // The function you write should return
 // the object passed in as an argument or the object created
+
 var addMethod = function addMethod(propertyName, method, obj) {
 
+  // so I can write '!obj' and JS knows that to check if that parameter
+  // was passed in? Or does this work because we are intending 'obj'
+  // to be referencing an object?
+
+  // are we using the word 'obj' as a semantic name for an object because
+  // we're intending to use that parameter as an object? We could just
+  // as easily use 'mary' for 'obj' as long as we know intent, yes?
```

Comment:
Yes, we can name it anything. What we name a parameter has no bearing on what is actually passed in. We *choose* to name things in ways that are helpful to our comprehension.